index.html

```diff
@@ -0,0 +1,128 @@
+          <li><a href="#Projects">Projects</a></li>
+          <li><a href="#Contact" >Contact Us</a></li>
+        </ul>
+        <a rel="noopener" href="https://github.com/zainabbehzad" id="profile-link" target="_blank">Subscribe</a>
+      </nav>
+    </div>
+    <div id="welcome-section">
+      <h4>Hello, my name is</h4>
+      <h1>Zainab<span> Behzad</span></h1>
+      <h3>I'm a Web Developer.</h3>
+      <div class="newslatter">
+        <form>
+          <input type="email" name="email" id="email" placeholder="Enter your Email">
+          <input type="submit" name="submit" value="Lets Start">
+        </form>
+      </div>
+    </div>
+    <section class="about">
+      <div class="main">
+        <div class="about-content">
+          <img src="images/b28fec1e-98ae-4934-b470-e556ac45f0df.jpeg"/>
+          <div class="about-text">
+            <h2 id="Me">About Me</h2>
+            <h5>Web <span class="red">Developer</span></h5>
+            <p> I am a front-end web developer. I can provide clean code and pixel perfect design. I also make the website more and more interactive with web animations.A responsive design makes your website accessable to all users, regardless of their device. </p>
+            <button type="button">Lets Talk</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="skills" class="skills">
+      <h2>My <span class="red">Skills</span></h2>
+      <div class="skills-container">
+        <div class="skill-item">
+          <h3>HTML</h3>
+          <p>Proficient</p>
+          <i class="fab fa-html5"></i>
+        </div>
+        <div class="skill-item">
+          <h3>CSS</h3>
+          <p>Proficient</p>
+          <i class="fab fa-css3-alt"></i>
+        </div>
+      </div>
+    </section>
+    <hr>
+    <section class="about">
+      <div class="main">
+        <div class="about-content">
+          <img src="images/Screenshot (19).png" />
+          <div class="about-text">
+            <h2 id="Project">About Projects</h2>
+            <h5>Linters <span class="red">Config</span></h5>
+            <p>Linters are tools that analyze code for potential errors, bugs, stylistic inconsistencies, and other issues.</p>
+            <a href="https://zainabbehzad.github.io/First_project/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button type="button">Source</button></a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="projects" class="projects">
+      <h2 id="Projects">Some of my Projects</h2>
+      <div class="project-grid">
+        <div class="project-item">
+          <img class="project-img" src="./images/Screenshot (15).png">
+          <div class="buttons">
+            <a href="https://zainabbehzad.github.io/survey-form/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button class="button">Source</button></a>
```

Comment:
- You linked the source code to your GitHub account, instead, add the exact link of this project.